applications/callflow/src/module/cf_kvs_set.erl

```diff
@@ -0,0 +1,143 @@
+%%%-------------------------------------------------------------------
+
+%%%-------------------------------------------------------------------
+-module(cf_kvs_set).
+
+-export([handle/2]).
+-export([get_kv/2, get_kv/3]).
+-export([kvs_to_props/1]).
+-export([format_json/1]).
+
+-include("../callflow.hrl").
+
+-define(KVS_DB, <<"kvs_collections">>).
+-define(COLLECTION_KVS, <<"Custom-KVS">>).
+-define(COLLECTION_MODE, <<"KVS-Mode">>).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec handle(wh_json:object(), whapps_call:call()) -> 'ok'.
+handle(Data, Call) ->
+    Call2 = set_kvs(Data, Call),
+    cf_exe:set_call(Call2),
+    cf_exe:continue(Call2).
+
+get_kv(Key, Call) ->
+    get_kv(?COLLECTION_KVS, Key, Call).
+
+get_kv(Collection, Key, Call) ->
+    wh_json:get_value(Key, get_collection(Collection, Call)).
+    
+set_kvs(Data, Call) ->
+    Call2 = set_kvs_mode(wh_json:get_value(<<"kvs_mode">>, Data, undefined), Call),
+    Data2 = wh_json:delete_key(<<"kvs_mode">>, Data),
+    Keys = wh_json:get_keys(Data2),
+    lists:foldl(fun(Key, Call3) ->
```

Comment:
Could you move the `fun` into a named private function? Like `fold_appropriate_name_here/`
This way if code breaks here we get a useful name instead of `#Fun<123123.123122.232>`